### PR TITLE
Adding a new method to fix #199

### PR DESCRIPTION
**Pull request by maguero**
Combo call to create subscriptions with add-ons.

### CODE REVIEW

File: src/main/java/org/killbill/billing/entitlement/api/EntitlementSpecifier.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+package org.killbill.billing.entitlement.api;
+
+import java.util.List;
+
+import org.killbill.billing.catalog.api.PlanPhasePriceOverride;
+import org.killbill.billing.catalog.api.PlanPhaseSpecifier;
+
+/**
+ * ...
```
Comment (pierre) [inline src/main/java/org/killbill/billing/entitlement/api/EntitlementSpecifier.java]: Remove empty documentation.

File: src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java [hunk 2]
```diff
@@ -59,6 +59,22 @@ public Entitlement createBaseEntitlement(UUID accountId, PlanPhaseSpecifier spec
             throws EntitlementApiException;
 
     /**
+     * Create a new base entitlement and addOn entitlements for that account.
+     * <p/>
```
Comment (pierre) [inline src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java]: This will unfortunately break on JDK 1.8. You need to use `<p>` instead of `<p/>`.

Comment (maguero) [inline src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java]: There are other existing methods using <p/>. Should I also fix them?

Comment (pierre) [inline src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java]: Ah, indeed :cry: 

Let's do it in another PR. I just ran `mvn javadoc:javadoc` and there are a lot of other issues besides this (such as missing descriptions).

**Comment by sbrossie** [comment]
:+1: Looks good.
